include/ttmlir-c/TTAttrs.h

```diff
@@ -7,6 +7,7 @@
 
 #include "mlir-c/AffineMap.h"
 #include "ttmlir-c/Dialects.h"
+#include <sys/types.h>
```

Comment:
Can we replace with `#include <cstdint>` and replace `ushort` with `std::uint16_t`

Reply:
Should we replace ushort with std::uint16_t for all enums? Or just for supportedDataTypes? So should I also change /tt-mlir/include/ttmlir/Target/Common/types.fbs and any other file where ushort is used?

Reply:
My preference is to just change DataType enum and leave others alone for future cleanup task/PR. With change to use uint16 in types.fbs for this enum, probably don't even need the extra include of header here (since I see references to uint32_t already).